src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -105,17 +105,32 @@
 					:disabled="disabled"
 					@recording="handleRecording"
 					@audio-file="handleAudioFile" />
-
-				<Button v-else
-					:disabled="disabled"
-					type="tertiary"
-					native-type="submit"
-					:aria-label="t('spreed', 'Send message')"
-					@click.prevent="handleSubmit">
-					<Send title=""
-						:size="16"
-						decorative />
-				</Button>
+				<!-- Send buttons -->
+				<template v-else>
+					<Actions :force-menu="true">
+						<!-- Silent send -->
+						<ActionButton :close-after-click="true"
+							icon="icon-upload"
+							:title="t('spreed', 'Send silently')"
+							@click.prevent="handleSubmit({ silent: true })">
+							{{ t('spreed', 'The recipients won\'t be notifyed about this message') }}
```

Comment:
```suggestion
							{{ t('spreed', 'The recipients won\'t be notified about this message') }}
```

Also maybe we put this in a computed and for one to one chats we then use:
```suggestion
							{{ t('spreed', 'The recipient won\'t be notified about this message') }}
```

Other than that should it be "participants?